src/test/java/org/sonarlint/intellij/issue/IssueManagerTest.java

```diff
@@ -45,13 +46,12 @@
 import static org.mockito.Mockito.verify;
 import static org.mockito.Mockito.when;
 
-public class IssueManagerTest extends SonarTest {
+public class IssueManagerTest extends LightPlatformCodeInsightFixture4TestCase {
```

Comment:
Why use the parent class instead of our `AbstractSonarLintLightTest` here? IMHO, sticking to our own test parent can help with migrations in the future.

Reply:
:+1: This is a mistake.